ArcGISRuntimeSDKQt_CppSamples/Geometry/FormatCoordinates/FormatCoordinates.cpp

```diff
@@ -179,8 +179,7 @@ void FormatCoordinates::setMapView(MapQuickView* mapView)
 
     // using the point, refresh the graphic and the text
     handleLocationUpdate(std::move(mapPoint));
-  },
-  Qt::UniqueConnection); // lambda-unique-connection: Finds usages of Qt::UniqueConnection when the slot is a functor, lambda or non-member function. That connect() overload does not support Qt::UniqueConnection.
+  });
```

Comment:
As indicated I'd like the samples team to follow up on this. Removing it is good since it won't show the clazy code anymore, and it's not working now anyways.

cc: @tanneryould.

Reply:
No. There is nuance to this. `Qt::UniqueConnection` does not work with lambda style connections, so the task is to:
 * Determine if a UniqueConnection is actually needed here. Is there any chance this will be connected multiple times?
 * If that won't happen, remove `Qt::UniqueConnection` since it does not do anything here anyways.
 * If this will be connected more than once, we need to update the sample code to use some other logic to make sure this connection is not made more than once.

In the short term, it is safe to remove `Qt::UniqueConnection` now since it won't work as written.